src/data/mv_polynomial.lean

```diff
@@ -815,13 +810,12 @@ end comm_ring
 section rename
 variables {α} [comm_semiring α] [decidable_eq α] [decidable_eq β] [decidable_eq γ] [decidable_eq δ]
 
-def rename_hom (f : β → γ) : mv_polynomial β α →+* mv_polynomial γ α :=
-@ring_hom.of _ _ _ _ (eval₂ C (X ∘ f)) (eval₂.is_semiring_hom C (X ∘ f))
+def rename (f : β → γ) : mv_polynomial β α → mv_polynomial γ α :=
```

Comment:
It seems like we're jumping back and forth between bundled and unbundled homs... if we revert this, I think we should clearly record why.

Reply:
`rename_hom` was not in the `master`. I added this but it made merging `master` harder, so I removed it again. Possibly, I'll post another PR with `rename_hom`.